docs/api.rst

```diff
@@ -5,7 +5,7 @@ API
     :noindex:
     :synopsis: public Jinja API
 
-This document describes the API to Jinja and not the template language.  It
+This document describes the API to Jinja and not the template language (for that, see `here <https://jinja.palletsprojects.com/en/2.11.x/templates/>`_.  It
```

Comment:
This needs to be a reference to the docs page, not a direct link to the hosted docs.

```rst
for that, see :docs:`/templates`
```